spartan/aztec-network/templates/_helpers.tpl

```diff
@@ -153,6 +129,8 @@ Service Address Setup Container
       value: "{{ .Values.network.public }}"
     - name: NAMESPACE
       value: {{ .Release.Namespace }}
+    - name: OTEL_COLLECTOR_ENDPOINT
```

Comment:
Otel collector port is not provided here although it is available for use in the script above?

Reply:
I guess we want to keep it as the default

Reply:
yeah I wanted to make it available potentially in future but it is a bit misleading. You can change the port by changing the full URL but no real reason to change ports imo